### DISCUSSION TITLE
Scenario "Santiago": Find the secret combination

### DISCUSSION THREAD

**Discussion by fduran** [SadServers/sadservers#19] [Scenarios]
Discussions about the [Santiago scenario](https://github.com/fduran/sadservers/tree/main/scenarios/santiago)

**Comment by mra-dk** [comment]
Feedbacks: Thankyou for the nice way to crunch linux solutions. Had a problem with this one.
Checked the correct number into the /home/admin/solution, but getting the wrong md5sum ?

**Comment by fduran** [comment]
hmm, check /home/admin/agent/check.sh for the actual "Check My Solution" test. It's also possible there's an issue with a new line termination. `echo 1234 > /home/admin/solution` should work (for a "1234" solution)

**Comment by cristianSysadmin** [comment]
Hi, I`m new with this web but I tried the solution of the fist part and I have this result
admin@i-044a69a080d3a5f5b:~$ grep -rc Alice /home/admin/*.txt
11-0.txt:398
1342-0.txt:1
1661-0.txt:12
84-0.txt:0

Your solution is 411... How?

**Reply by fduran** [reply]
398 + 1 + 12 = 411 will be the total

I agree this is [confusing](https://github.com/fduran/sadservers/issues/10)

**Comment by vinnishtein** [comment]
Hello, it is unclear what combination of numbers i have to enter:

```
# grep Alice /home/admin/*.txt -c
/home/admin/11-0.txt:398
/home/admin/1342-0.txt:1
/home/admin/1661-0.txt:12
/home/admin/84-0.txt:0

# grep Alice /home/admin/1342-0.txt -A 1
                                Alice
                        156 CHARING CROSS ROAD

```
sum of first numbers and second number?
(398+1+12) 156 ?
411 156 ?
first number and second number ?
398 156 ?
Checked different variants, withot spaces, without newlines, can't pass
